.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,39 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2.0.0"
+
+      - name: "Install PHP with extensions"
+        uses: "shivammathur/setup-php@1.8.1"
+        with:
+          coverage: "none"
+          extensions: "simplexml"
+          php-version: "7.0"
+
+      - name: "Cache dependencies installed with composer"
```

Comment:
our repo does not have external dependencies.. we can drop this

Reply:
We do now - I required `friendsofphp/php-cs-fixer`!